extensions/amp-story/1.0/amp-story.css

```diff
@@ -25,6 +25,10 @@ amp-story, amp-story-page, amp-story-grid-layer, amp-story-cta-layer {
   overflow: hidden !important;
 }
 
+[dir=rtl] {
+  text-align: right !important;
```

Comment:
This is already the default value.
I think it'd be better to remove this, and override it wherever we forced `text-align: left;`